enterprise/frontend/src/metabase-enterprise/metabot/context.tsx

```diff
@@ -36,13 +39,24 @@ export const MetabotProvider = ({
   const providerFnsRef = useRef<Set<ChatContextProviderFn>>(new Set());
   const store = useStore();
 
+  const [listDbs] = useLazyListDatabasesQuery();
+
   const getChatContext = useCallback(async () => {
```

Comment:
Glad I made this async a while back even though we didn't end up needing it at the time!